tests/settings_test.py

```diff
@@ -17,8 +19,8 @@
 default_duration = 45
 
 [main]
-assetdir = /home/pi/screenly_assets
-database = /home/pi/.screenly/screenly.db
+assetdir = """ + home + """/screenly_assets
```

Comment:
Let's make this a bit more pythonic.

`assetdir = "{}/screenly_assets".format(user_home_dir)`